controllers/users/users.js

```diff
@@ -60,3 +60,9 @@ exports.registerUser = (req, res) => {
     }
   });
 };
+
+exports.deleteUser = (req, res) => {
+  User.findOneAndRemove({ _id: req.user.id })
```

Comment:
we should remove user from db or mark as deleted with some field?

Reply:
we can discuss it and change later